src/main/java/com/cloudbees/plugins/credentials/CredentialsDescriptor.java

```diff
@@ -388,7 +388,7 @@ public static ModelObject lookupContext(String provider, String token) {
                     if (pathSegments.size() >= 2) {
                         String firstSegment = pathSegments.get(0);
                         if ("user".equals(firstSegment)) {
-                            User user = User.get(pathSegments.get(1));
+                            User user = User.getById(pathSegments.get(1), false);
```

Comment:
```suggestion
                            User user = User.getById(pathSegments.get(1), true);
```

to preserve non-null semantics of original call